addons/udes_delivery_control/models/stock_picking.py

```diff
@@ -166,6 +252,13 @@ def _compute_show_mark_as_todo(self):
         for picking in self.filtered("u_is_delivery_control"):
             picking.show_mark_as_todo = picking.state == "draft"
 
+    @api.onchange("u_loading_type")
+    def _compute_loading_type(self):
+        """Compute is_backloading and is_unloading from u_loading_type"""
+        for record in self:
+            record.is_unload = "unload" in record.loading_type
```

Comment:
Shouldn't this be `u_loading_type`? If so I will resolve my comment about refactoring the logic as they are different field names.

Reply:
Errr! I made a mistake. Good eyes mate :+1: 
Changed